hammerdb/sql/tpcc-distribute-funcs.sql

```diff
@@ -1,14 +1,3 @@
-
--- THESE SHOULD BE UNCOMMENTED IF STORED PRODECURES ARE FALSE IN CONFIGS
```

Comment:
I guess this is because now we distribute the functions automatically, right ?

But what happens if we want to test against older versions of citus ?

Reply:
I added the comment back